### DISCUSSION TITLE
create manifest with model_db=None generates error

### DISCUSSION THREAD

**Discussion by oMersa344** [micom-dev/micom#98] [Q&A] [answered]
I'm tring to build a model of only one yeast using the iMM904 model.
The only reason I want to use the `build` function and create the manifest in such a case is to use later the `fix_medium` and `candidate_medium`, which cannot work with a specific model (I've tried to build one using `Community`).

The input DataFrame looks like this:
```
    sample_id   id  abundance                 file
0  sample1  taxa1   0.999999  /data/model.taxa1.xml.gz
```
When calling the function `build` to create the manifest, I used `None` as the argument for the `model_db`  because I have the file column in the data set: 
`manifest = build(data, out_folder="out_models", model_db=None, cutoff=0.0001, threads=2)`

The above `build` command generates the following error:
```
Traceback (most recent call last):
  File "USERS\MEAT\find_minimal_med.py", line 14, in <module>
    manifest = build(data, out_folder="out_models", model_db=None, cutoff=0.0001, threads=2)
  File "D:\Program Files\conda\envs\MEAT\lib\site-packages\micom\workflows\build.py", line 103, in build
    out_path = pd.Series({s: os.path.join(out_folder, s + ".pickle") for s in samples})
  File "D:\Program Files\conda\envs\MEAT\lib\site-packages\micom\workflows\build.py", line 103, in <dictcomp>
    out_path = pd.Series({s: os.path.join(out_folder, s + ".pickle") for s in samples})
numpy.core._exceptions._UFuncNoLoopError: ufunc 'add' did not contain a loop with signature matching types (dtype('int64'), dtype('<U7')) -> None
```
I wonder what is the cause for this error, any suggestions?

**Comment by cdiener** [comment, marked as answer]
Hey is that an example or the actual DataFrame you used? It seems to complain that your `sample_id` is not a string, but in your example it is one.

**Reply by oMersa344** [reply]
Hey, I absolutely missed it. It works now. 
Thanks 🙏🏽
